components/blocks/Landing/Landing.vue

```diff
@@ -12,8 +12,8 @@ const leaderboards = await useGetLeaderboards()
   <div>
     <h2 class="mb-4 text-3xl font-bold">Games</h2>
     <div class="flex flex-col lg:flex-row">
-      <Loader v-if="leaderboards.loading" />
-      <LandingLeaderboards :leaderboards="leaderboards.data || []" />
+      <Loader v-if="leaderboards.loading"></Loader>
+      <LandingLeaderboards :leaderboards="leaderboards.data?.data || []" />
```

Comment:
Our list endpoints return this object instead of arrays of the entity now:
```
{
    data: <array_of_entities>,
    total: int,
    limitDefault: int,
    limitMax: int
}